### PR TITLE
Fixed #41: css modified to make the footer stick to the bottom

### LINKED ISSUE #41: stick the footer to the bottom
if the content is less footer is going up so make the footer to stick the bottom
[example site](https://sch00lb0y.github.io/tempCheck/#)

### PR DESCRIPTION
**Pull request by sch00lb0y**
I have added min-height for the content div tag to stick the footer to the bottom.
close #41 . @niranjan94 @pri22296 @imujjwal96 please review .
here is the [demo](https://sch00lb0y.github.io/tempCheck/)

### CODE REVIEW

**Comment by imujjwal96** [comment]
image
There is still some gap

**Comment by niranjan94** [comment]
@sch00lb0y don't push the footer down by using a div inside body. Try this 
https://getbootstrap.com/examples/sticky-footer/

Works really well for bootstrap sites in all cases.

**Comment by sch00lb0y** [comment]
@niranjan94 i have changed that. it works well. Thanks for your suggestion. here is the updated [demo](https://sch00lb0y.github.io/tempCheck/). please check

**Comment by sch00lb0y** [comment]
@niranjan94 @pri22296 @imujjwal96 please review this.
Thanks

**Comment by pri22296** [comment]
@sch00lb0y I don't know much about front-end. But if it helps, footer does not stick on small screens such as mobile, or if we minimize the browser too much.

File: fossasia/static/custom.css [hunk 1]
```diff
@@ -2,6 +2,7 @@ body{
   font-family: 'Open Sans', "Helvetica Neue", Helvetica, Arial, sans-serif;
   background: #fff;
   color: #777;
+  margin-bottom: 60px;
```
Comment (niranjan94) [inline fossasia/static/custom.css]: You need to set this to the same height as the footer height. If footer height is 100px, this margin should also be 100px;

File: fossasia/static/custom.css [hunk 2]
```diff
@@ -176,3 +177,10 @@ footer.classic .social-profiles {
 .menu li a:hover {
   border-bottom: 2px solid #fff;
 }
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
```
Comment (niranjan94) [inline fossasia/static/custom.css]: Your footer height seems to be around 185px; for it to work properly on all screens.

**Comment by imujjwal96** [comment]
If you remove the height specified in .footer class, the bug fixes.

**Comment by sch00lb0y** [comment]
screen shot 2017-05-27 at 23 05 36
screen shot 2017-05-27 at 11 44 23
i have made changes here is the sample [site](https://sch00lb0y.github.io/tempCheck/)
and also i have uploaded the screenshots . @niranjan94 @imujjwal96 @pri22296 please check Thanks

**Pull request merged**